ui/src/components/FileExplorer.vue

```diff
@@ -942,6 +942,11 @@ export default {
         this.page = this.page_count
       }
       this.sortChanged()
+      const allowedExtensions = ['.bmp', '.png', '.tiff', '.jpg']
```

Comment:
There is a configuration setting that defines the file extensions of the images. We could link this instead and make this constant configurable then.